app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import { Button, Modal } from 'react-bootstrap';
+import InfoTooltip from '../../components/util/InfoTooltip';
+import { shallow, mount } from 'enzyme';
+import CloseContact from '../../components/close_contact/CloseContact.js'
+import { mockPatient1 } from '../mocks/mockPatients'
+import { mockCloseContact0, mockCloseContact1 } from '../mocks/mockCloseContact'
```

Comment:
Is there a reason we started counting at 0 versus 1? I think everyone else starts at 1 so we should be consistent. 

Also super nitpicky - but can you organize these headers like the rest of the of the test files so it's all in the same order?

Reply:
Resolved with latest commit.